src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -1917,7 +1917,7 @@ private SSHConnectionInfo()
             this.UserName = userName;
             this.ComputerName = computerName;
             this.KeyFilePath = keyFilePath;
-            this.Port = DefaultPort;
+            this.Port = ReservedPort;
```

Comment:
I think the term 'ReservedPort' is confusing.  We can remove it and let the `Port` value be initialized to zero automatically, and add comment that either default port 22 or configured port will be used.

Reply:
I believe it's referring to port 0 being a reserved port: https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml

Reply:
Nevertheless I don't feel we need a named const set to zero for this.  The code should also make it clear (via comments) that SSH port behavior is to default to 22 or a configured value, to help the next developer maintaining the code.

Reply:
I've removed ReservedPort.